torchvision/models/detection/rpn.py

```diff
@@ -157,7 +153,8 @@ def forward(self, image_list, feature_maps):
         # type: (ImageList, List[Tensor])
         grid_sizes = list([feature_map.shape[-2:] for feature_map in feature_maps])
         image_size = image_list.tensors.shape[-2:]
-        strides = [[int(image_size[0] / g[0]), int(image_size[1] / g[1])] for g in grid_sizes]
+        strides = [[torch.scalar_tensor(image_size[0] / g[0], dtype=torch.int64),
```

Comment:
is `scalar_tensor` different than `tensor` with a scalar value?

Reply:
The only difference is that it only holds scalars. We want to be consistent in using scalar tensors for all dynamically traced scalars.

Reply:
Given that `scalar_tensor` is not part of the public API of PyTorch (and it's not documented), would it give different results / be a problem if we use `tensor` instead?